packages/cli/src/build.ts

```diff
@@ -3,7 +3,7 @@ import co from '@cobalt-ui/core';
 import fs from 'node:fs';
 import {URL} from 'node:url';
 
-export async function build(rawSchema: Group, config: ResolvedConfig): Promise<ParseResult> {
+export default async function build(rawSchema: Group, config: ResolvedConfig): Promise<ParseResult> {
```

Comment:
No reason for this change other than simplification. 🤔 IIRC I was on a very short-lived “no default exports” kick but I don’t remember why now. I never wrote any reason down for this.